pkg/apis/core/v1alpha1/constants/types_constants.go

```diff
@@ -209,6 +209,9 @@ const (
 	// "core.gardener.cloud/apiserver-exposure" to indicate that it's responsible for apiserver exposure (via SNI).
 	LabelAPIServerExposureGardenerManaged = "gardener-managed"
 
+	// LabelWorkerPoolSystemComponentsAllow is a constant that indicates whether the worker pool should host system components
+	LabelWorkerPoolSystemComponentsAllow = "worker.gardener.cloud/system-components"
```

Comment:
```suggestion
	LabelWorkerPoolSystemComponents = "worker.gardener.cloud/system-components"
```
Please also adjust doc string.